README.md

```diff
@@ -11,6 +11,25 @@ It's very simple!
 
 # Electrical
 
+The LX-* servos all use a 3.3v driven bi directional asynchronus serial. It is similar to UART, but uses both signels on one pin. Because of this, the Master TX line has to be connected only while transmitting. THe correct way to do this is a Buffer chip. 
```

Comment:
The buffer chip is only required to interface between 3.3v and 5v and/or to increase drive current, the MCU pins are totally capable of tri-state.

Reply:
I have spent a few days with a set of 2 of the LX-16 and LX-15's and my o-scope. A few things are clear, the motors drive 3.3v serial, so the only level shifting needed would be *DOWN* to 3.3 if you are on a 5v chip. 

This comment did give me an idea, try to use the code to set the pin to High Z when not transmitting...

Reply:
The original code puts the TX pin in open-drain mode, no need for tri-state.

Reply:
I am working on a Arduino generic version, and trying to rely less on chip specific implementations. the tristate for each pin is not well handled by Arduino when also using a serial port.